fe/fe-core/src/test/java/com/starrocks/credential/CloudConfigurationFactoryTest.java

```diff
@@ -33,10 +37,174 @@ public void testBuildCloudConfigurationForTabular() {
         CloudConfiguration cloudConfiguration = CloudConfigurationFactory.buildCloudConfigurationForTabular(map);
         Assert.assertNotNull(cloudConfiguration);
         Assert.assertEquals(CloudType.AWS, cloudConfiguration.getCloudType());
-        Assert.assertEquals("AWSCloudConfiguration{awsCloudCredential=AWSCloudCredential" +
-                "{useAWSSDKDefaultBehavior=false, useInstanceProfile=false, " +
-                "accessKey='ak', secretKey='sk', sessionToken='token', iamRoleArn='', " +
-                "externalId='', region='region', endpoint=''}, enablePathStyleAccess=false, " +
-                "enableSSL=true}", cloudConfiguration.getCredentialString());
+        Assert.assertEquals(
+                "AWSCloudConfiguration{resources='', jars='', cred=AWSCloudCredential{" +
+                        "useAWSSDKDefaultBehavior=false, useInstanceProfile=false, " +
+                        "accessKey='ak', secretKey='sk', sessionToken='token', iamRoleArn='', " +
+                        "externalId='', region='region', endpoint=''}, enablePathStyleAccess=false, " +
+                        "enableSSL=true}", cloudConfiguration.toConfString());
+    }
+
+    @Test
+    public void testAWSCloudConfiguration() {
+        Map<String, String> map = new HashMap<String, String>() {
+            {
+                put(CloudConfigurationConstants.AWS_S3_ACCESS_KEY, "XX");
+                put(CloudConfigurationConstants.AWS_S3_SECRET_KEY, "YY");
+                put(CloudConfigurationConstants.AWS_S3_REGION, "ZZ");
+            }
+        };
+        CloudConfiguration cc = CloudConfigurationFactory.buildCloudConfigurationForStorage(map);
+        Assert.assertEquals(cc.getCloudType(), CloudType.AWS);
+        TCloudConfiguration tc = new TCloudConfiguration();
+        cc.toThrift(tc);
+        Configuration conf = new Configuration();
+        cc.applyToConfiguration(conf);
+        cc.toFileStoreInfo();
+        System.out.println(cc.toConfString());
```

Comment:
Test about `enable_path_style_access` and `enable_ssl` either.

And using EXECPTED_EQ to check string is correct.

Reply:
fixed.